pkg/kudoctl/util/repo/repo.go

```diff
@@ -0,0 +1,42 @@
+package repo
+
+// RepositoryConfiguration represents a collection of parameters for framework repository
+type RepositoryConfiguration struct {
+	//LocalPath string `json:"localPath"`
+	URL string `json:"url"`
+}
+
+// NewRepositoryConfiguration returns a given RepositoryConfiguration struct
+func NewRepositoryConfiguration() *RepositoryConfiguration {
+	return &RepositoryConfiguration{
+		URL: "https://kudo-repository.storage.googleapis.com",
+	}
+}
+
+// Metadata for a Framework. This models the structure of a bundle.yaml file.
+type Metadata struct {
+	// The name of the framework
+	Name string `json:"name,omitempty"`
+	// A SemVer 2 conformant version string of the framework
+	Version string `protobuf:"bytes,4,opt,name=version" json:"version,omitempty"`
+	// The URL to a relevant project page, git repo, or contact person
+	Home string `json:"home,omitempty"`
+	// Source is the URL to the source code of this framework
+	Sources []string `json:"sources,omitempty"`
+	// A one-sentence description of the framework
+	Description string `json:"description,omitempty"`
+	// A list of name and URL/email address combinations for the maintainer(s)
+	Maintainers []*Maintainer `json:"maintainers,omitempty"`
+	// Whether or not this framework is deprecated
+	Deprecated bool `json:"deprecated,omitempty"`
```

Comment:
would it make sense to have an extra field called `Tags []string` to e.g. to use `["stable", "production-1"]` ?

Reply:
One step at a time. Fine to me to have this removed and introduced back later on.

Reply:
Yeah I dropped some fields that felt like we can always add them in the future :)

Reply:
Let's remove this then.